widget.py

```diff
@@ -141,6 +144,9 @@ def join_multicast_group(self, ip_addr, port):
             self.ui.udpConnectButton.setText("Close UDP connection")
             self.ui.udpIpAddressInput.setReadOnly(True)
             self.ui.udpPortInput.setReadOnly(True)
+            file_name = QDateTime.currentDateTime().toString("yyyy-MM-dd_HH-mm")
+            file_name += '.dump'
+            self.file_out = open(file_name, "a+b")
```

Comment:
I'm good with the file naming, can we put these files in a recordings directory and add that to the gitignore?